classic/main.go

```diff
@@ -0,0 +1,170 @@
+package main
+
+import (
+	"context"
+	"math/big"
+	"path/filepath"
+	"strings"
+
+	"github.com/openrelayxyz/plugeth-utils/core"
+	"github.com/openrelayxyz/plugeth-utils/restricted"
+)
+
+var (
+	ClassicBootnodes = []string{
+
+		"enode://942bf2f0754972391467765be1d98206926fc8ad0be8a49cd65e1730420c37fa63355bddb0ae5faa1d3505a2edcf8fad1cf00f3c179e244f047ec3a3ba5dacd7@176.9.51.216:30355", // @q9f ceibo
+		"enode://0b0e09d6756b672ac6a8b70895da4fb25090b939578935d4a897497ffaa205e019e068e1ae24ac10d52fa9b8ddb82840d5d990534201a4ad859ee12cb5c91e82@176.9.51.216:30365", // @q9f ceibo
+		"enode://b9e893ea9cb4537f4fed154233005ae61b441cd0ecd980136138c304fefac194c25a16b73dac05fc66a4198d0c15dd0f33af99b411882c68a019dfa6bb703b9d@18.130.93.66:30303",
+	}
+
+	dnsPrefixETC string = "enrtree://AJE62Q4DUX4QMMXEHCSSCSC65TDHZYSMONSD64P3WULVLSF6MRQ3K@"
+
+	ClassicDNSNetwork1 string = dnsPrefixETC + "all.classic.blockd.info"
+
+	snapDiscoveryURLs []string
+
+	forkBlockIds = []uint64 {1150000, 2500000, 3000000, 5000000, 5900000, 8772000, 9573000, 10500839, 11700000, 13189133, 14525000}
+
+	forkTimeIds = []uint64{}
+)
+
+type ClassicService struct {
+	backend core.Backend
+	stack   core.Node
+}
+
+var (
+	pl      core.PluginLoader
+	backend restricted.Backend
+	log     core.Logger
+	events  core.Feed
+)
+
+var (
+	httpApiFlagName = "http.api"
+	MainnetFlag = "mainnet"
+	GoerliFlag = "goerli"
+	SepoliaFlag = "sepolia"
+	HoleskyFlag = "holesky"
+)
+
+func Initialize(ctx core.Context, loader core.PluginLoader, logger core.Logger) { 
+	pl = loader
+	events = pl.GetFeed()
+	log = logger
+	v := ctx.String(httpApiFlagName)
+	if v != "" {
+		ctx.Set(httpApiFlagName, v+",plugeth")
+	} else {
+		ctx.Set(httpApiFlagName, "eth,net,web3,plugeth")
+		
+	}
+
+	switch {
+		case ctx.Bool(MainnetFlag):
+			panic("This node is optimized to run the Ethereum Classic Network only")
```

Comment:
Maybe this message should suggest looking in the plugins folder, in case people can't figure out what they did to enable it?